tests/DocumentationExamplesTest.php

```diff
@@ -925,6 +925,41 @@ public function testExample_55_58()
         $this->assertInventoryCount(0);
     }
 
+    public function testExample_59_61()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+
+        // Start Changestream Example 1
+        $cursor = $db->inventory->watch();
+        $cursor->next();
+        $current = $cursor->current();
+        // End Changestream Example 1
+
+        $this->assertNull($current);
+
+        // Start Changestream Example 2
+        $cursor = $db->inventory->watch([], ['fullDocument' => \MongoDB\Operation\ChangeStreamCommand::FULL_DOCUMENT_UPDATE_LOOKUP]);
+        $cursor->next();
+        $current = $cursor->current();
+        // End Changestream Example 2
+
+        $this->assertNull($current);
+        $db->inventory->insertOne(['x' => '1']);
+        $cursor->next();
+        $current = $cursor->current();
```

Comment:
We can probably assert the change entry here as well. It will make not difference for example 3, since it's written defensively, but it makes the test itself more useful.